debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/CommitLogProcessor.java

```diff
@@ -102,15 +105,21 @@ void processCommitLog(File file) throws IOException {
             throw new IOException("Commit log " + file.getName() + " does not exist");
         }
         try {
-            LOGGER.info("Processing commit log {}", file.getName());
-            metrics.setCommitLogFilename(file.getName());
-            commitLogReader.readCommitLogSegment(commitLogReadHandler, file, false);
-            queue.enqueue(new EOFEvent(file, true));
-            LOGGER.info("Successfully processed commit log {}", file.getName());
+            try {
```

Comment:
Is this "inner" try/catch block really needed? Couldn't botch catch statements just be part of the same try/catch block?

Reply:
> Is this "inner" try/catch block really needed? Couldn't botch catch statements just be part of the same try/catch block?

Yes I think so, because the inner catch block which catches the `IOException` will also throw an `InterruptedException`.

```catch (IOException e) {
                queue.enqueue(new EOFEvent(file, false));
                LOGGER.warn("Error occurred while processing commit log " + file.getName(), e);
            }```

Reply:
Gotcha, makes sense.